.github/workflows/ci.yml

```
@@ -0,0 +1,69 @@
+        NODE_ENV: test
+        ACCUWEATHER_API_URL_FOR_KEY: ${{ secrets.ACCUWEATHER_API_URL_FOR_KEY }}
+        ACCUWEATHER_API_URL_FOR_INFO: ${{ secrets.ACCUWEATHER_API_URL_FOR_INFO }}
+        DATABASE_URL: postgres://${{ env.POSTGRES_USER }}:${{ env.POSTGRES_PASSWORD }}@localhost:5432/${{ env.POSTGRES_DB }}
+
+    # SonarCloud analysis
+    - name: SonarCloud Scan
+      uses: sonarsource/sonarcloud-github-action@master
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        project-base: ./
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)